contracts/StablecoinConverter.sol

```diff
@@ -167,4 +174,49 @@ contract StablecoinConverter is EpochTokenLocker {
         }
         revert("Price not provided for token");
     }
+
+    function documentTrades(
+        uint batchIndex,
+        address[] memory owner,  //tradeData is submitted as arrays
+        uint16[] memory orderId,
+        uint128[] memory volume,
+        uint128[] memory prices,
+        uint16[] memory tokenIdForPrice
+    ) internal {
+        previousSolutionBatchId = batchIndex;
+        uint len = owner.length;
+        for (uint i = 0; i < len; i++) {
```

Comment:
nit: why this len assignment? It's only used once.

Reply:
I did the gas test. it saved 11 gas :D 

You are right. I will change it.